tests/pom.xml

```diff
@@ -37,8 +37,8 @@
         <!--
         <module>dozer-osgi-tests-model</module>
         <module>dozer-osgi-tests</module>
-        -->
         <module>dozer-wildfly-tests</module>
```

Comment:
excluded this module because it is straightforward to migrate to new version of wildfly to support jdk 11 and Java EE 9